Dev-Corner/slides/toolshed.html

```diff
@@ -0,0 +1,861 @@
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Workflow
+                            <img src="../images/workflowRepDep.png" class="no-border" >
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ###  Workflow
+                            <img src="../images/workflowRepDep2.png" class="no-border" >
+                            - **Everything must be on the same toolshed**
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ## Tips
+                            ### What about tools working with *.loc files or new datatypes
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Tools with *.loc files or new datatypes
+                            - add them to the Tool Shed repository.
+
+                            - \*.loc files and datatypes files should be provided as *.sample file in a `tool-data` directory.
+
+                            - tool-data/*.loc.sample files come along with `tool_data_table_conf.xml.sample`.
+                            Their content or the file themselves are put to the right location on tool set up.
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Tools with *.loc files or new datatypes
+                            <img src="../images/repWithLoc_ss.png" class="no-border" width="60%">
+                            - <span style="font-size:60%">`tool_data_table_conf.xml.sample`</span>
+                            ```
+                            <tables>
+                            	<!-- Locations of indexes in the Bowtie2 mapper format -->
+                            	<table name="bowtie2_indexes" comment_char="#">
+                            		<columns>value, dbkey, name, path</columns>
+                            		<file path="tool-data/bowtie2_indices.loc" />
+                            	</table>
+                            </tables>
+                            ```
+                            <span style="font-size:60%">On install, the content of the file and the right
+                            path to the *.loc file is added to `shed_tool_data_table_conf.xml`</span>
+                        </script>
+                    </section>
+                </section>
+                <section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ## 4. Publication with Planemo
```

Comment:
Can you use this picture http://bgruening.github.io/training-material/Dev-Corner/slides/tool_integration.html#/6/3 to fit with the tool_integration part?